unicodetools/data/ucd/dev/PropertyAliases.txt

```diff
@@ -174,6 +174,8 @@ IDS                      ; ID_Start
 IDSB                     ; IDS_Binary_Operator
 IDST                     ; IDS_Trinary_Operator
 IDSU                     ; IDS_Unary_Operator
+ID_Compat_Math_Continue  ; ID_Compat_Math_Continue
+ID_Compat_Math_Start     ; ID_Compat_Math_Start
```

Comment:
Question for @Ken-Whistler @macchiati @asmusf: How should this file be sorted? By short or long name? ASCII or UCA or something else? Underscore -- ignore, by code point, like space, ...?

I would move these two to before ID_Continue, but I don't know if it matters.

Reply:
I agree they should be ahead of ID_Continue. In general, for this kind of file, I'd treat the underscores like spaces for ordering.